src/main/kotlin/lotto/Lotto.kt

```diff
@@ -1,31 +1,21 @@
 package lotto
 
-class Lotto {
+class Lotto(private val number: Set<Int>) {
```

Comment:
```suggestion
class Lotto(private val numbers: Set<Int>) {
```

단일 Int인 number 값으로 오해할  수 있겠네요~